src/Symfony/Bundle/SecurityBundle/Tests/Functional/AutowiringTypesTest.php

```diff
@@ -22,6 +22,15 @@ public function testAccessDecisionManagerAutowiring()
         $this->assertInstanceOf('Symfony\Component\Security\Core\Authorization\AccessDecisionManager', $accessDecisionManager);
     }
 
+    public function testDebugAccessDecisionManagerAutowiring()
+    {
+        static::bootKernel(array('debug' => true));
+        $container = static::$kernel->getContainer();
+
+        $accessDecisionManager = $container->get('test.autowiring_types.autowired_services')->getAccessDecisionManager();
+        $this->assertInstanceOf('Symfony\Component\Security\Core\Authorization\AccessDecisionManager', $accessDecisionManager);
```

Comment:
When in debug, it should be an instance of `Symfony\Component\Security\Core\Authorization\DebugAccessDecisionManager`